Svc/ActiveRateGroup/ActiveRateGroupImpl.cpp

```diff
@@ -43,6 +45,10 @@ namespace Svc {
         ActiveRateGroupComponentBase::init(queueDepth,instance);
     }
 
+    void ActiveRateGroupImpl::silenceSlipWarning() {
```

Comment:
The way we typically configure components is by using config headers (Ex: [FileDownlink](https://github.com/nasa/fprime/blob/devel/config/FileDownlinkCfg.hpp)) However, this config is per component type, not per component instance. Are you planning on silencing slip on a per-ActiveRateGroup instance basis?